website/docusaurus.config.js

```diff
@@ -213,13 +208,17 @@ ${content}`, // <-- this last part adds in the rest of the content, which would
           className: 'header-github-link',
           'aria-label': 'GitHub repository',
         },
-        { to: 'blog', label: 'Dev Changelog 🎉', position: 'right' },
+        { to: 'blog', label: 'Changelog 🎉', position: 'right' },
+        {
+          href: "login",
+          position: "right"
+        },
       ],
     },
     image: 'img/near_logo.svg',
     footer: {
-      links: [], 
-      copyright: 'Copyright © 2021 NEAR Protocol',
+      links: [],
+      copyright: "Copyright © 2023 NEAR Protocol",
```

Comment:
We'll need to implement lint rules docs around quotes as I've noticed our respective settings with quotations have been conflicting.